benchmark_suite.py

```
@@ -0,0 +1,315 @@
+                check_start = time.time()
+                
+                while time.time() - check_start < timeout:
+                    try:
+                        sock = socket.socket(socket.AF_INET, socket.SOCK_STREAM)
+                        sock.settimeout(0.1)
+                        result = sock.connect_ex(('127.0.0.1', 1256))
+                        sock.close()
+                        if result == 0:
+                            server_ready = True
+                            break
+                    except:
```

Ruff (0.11.9)

Do not use bare `except`

(E722)